extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -141,15 +141,23 @@ public class VertxHttpRecorder {
 
     public static final String GET = "GET";
     private static final Handler<HttpServerRequest> ACTUAL_ROOT = new Handler<HttpServerRequest>() {
+
+        /** JVM system property that disables URI validation, don't use this in production. */
+        private static final String DISABLE_URI_VALIDATION_PROP_NAME = "vertx.disableURIValidation";
```

Comment:
I used the `vertx` prefix to keep it consistent with `vertx.disableHttpHeadersValidation`, but we can certainly 
use `quarkus` instead if we think it's more appropriate

Reply:
Well we are the ones doing this validation, not Vert.x 🙂

Reply:
Sure, I meant that Vert.x should have validate that, not us 😉